Source/ProgressionSystemRuntime/Public/ProgressionSystemDataAsset.h

```diff
@@ -0,0 +1,58 @@
+﻿// Copyright (c) Valerii Rotermel & Yevhenii Selivanov
+
+#pragma once
+
+#include "ProgressionSystemDataAsset.generated.h"
+
+/**
+ * Contains all progression assets used in the module 
+ */
+UCLASS()
+class PROGRESSIONSYSTEMRUNTIME_API UProgressionSystemDataAsset : public UDataAsset
+{
+	GENERATED_BODY()
+
+public:
+	/** Returns the settings data asset. */
+	static const UProgressionSystemDataAsset& Get() { return *GetDefault<ThisClass>(); }
+
+	/** Returns the Progression Data Table
+	 * @see UProgressionSystemDataAsset::ProgressionDataTableInternal */
+	UFUNCTION(BlueprintPure, Category = "C++")
+	FORCEINLINE UDataTable* GetProgressionDataTable() const { return ProgressionDataTableInternal; }
+
+	/** Returns a progression menu widget to be displayed in the main menu*/
+	UFUNCTION(BlueprintPure, Category = "C++")
+	FORCEINLINE TSubclassOf<class UProgressionMenuWidget> GetProgressionMenuWidget() const { return ProgressionMenuWidgetInternal; }
+
+	/** Returns a progression save widget to be displayed in the end of game */
+	UFUNCTION(BlueprintPure, Category = "C++")
+	FORCEINLINE TSubclassOf<class UProgressionSaveWidget> GetProgressionSaveWidget() const { return ProgressionSaveWidgetInternal; }
+
+	/** Returns a locked progression icon reference */
+	FORCEINLINE TObjectPtr<class UTexture2D> GetLockedProgressionIcon() const { return LockedProgressionIconInternal; }
+
+	/** Returns a unlocked progression icon reference */
+	FORCEINLINE TObjectPtr<class UTexture2D> GetUnlockedProgressionIcon() const { return UnlockedProgressionIconInternal; }
```

Comment:
This and above Getters are missing `UFUNCTION(BlueprintPure, Category = "C++")`. Mak sure you have it for all properties.

Reply:
done